turbo.sql

```diff
@@ -0,0 +1,42 @@
+CREATE EXTENSION "uuid-ossp";
+
+CREATE TABLE "chat_rooms" (
+  "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
```

Comment:
@svkaizoku Honestly, I have just been using using UUID as a default ID ever since I saw sung using it.
But I guess this is good opportunity to evaluate UUID v SERIAL.

Reply:
Found a couple of interesting blogs about this.
https://thebuild.com/blog/2023/02/16/uuids-vs-serials-for-keys/
https://brandur.org/nanoglyphs/026-ids

Reply:
Lets discuss over call.